src/tiles/grid/item-tile.ts

```diff
@@ -49,6 +54,51 @@ export class ItemTile extends LitElement {
     `;
   }
 
+  private get doesSortedByDate() {
+    return ['date', 'reviewdate', 'addeddate', 'publicdate'].includes(
+      this.sortParam?.field as string
+    );
+  }
+
+  private get sortedDateInfoTemplate() {
+    let sortedValue;
+
+    switch (this.sortParam?.field) {
+      case 'date':
+        sortedValue = { field: 'published', value: this.model?.datePublished };
+        break;
+      case 'reviewdate':
+        sortedValue = { field: 'reviewed', value: this.model?.dateReviewed };
+        break;
+      case 'addeddate':
+        sortedValue = { field: 'added', value: this.model?.dateAdded };
+        break;
+      case 'publicdate':
+        sortedValue = { field: 'archived', value: this.model?.dateArchived };
+        break;
+      default:
+        break;
+    }
+
+    return html`
+      <div class="truncated date-sorted-by">
```

Comment:
a little nitpick i guess, can we place the className as `date-sorted-by` goes first followed by `truncated`?
doing so, it will be similar to the other parts where we add `truncated` className